jax/lax/lax.py

```diff
@@ -396,6 +409,7 @@ def conv_general_dilated(lhs, rhs, window_strides, padding, lhs_dilation=None,
       a 3-tuple `(lhs_spec, rhs_spec, out_spec)`, where each element is a string
       of length `n+2`.
     feature_group_count: integer, default 1. See XLA HLO docs.
+    precision: Optional. Either `None`, or a `Precision` enum value.
```

Comment:
Should we specify the behavior when passing `None` (and specify that it's the default)?

Reply:
Done.